protocol/synthetix/contracts/modules/usd/USDTokenModule.sol

```diff
@@ -83,7 +83,7 @@ contract USDTokenModule is ERC20, InitializableMixin, IUSDTokenModule {
 
         ERC20Storage.Data storage store = ERC20Storage.load();
 
-        if (amount < store.allowance[from][spender]) {
+        if (amount > store.allowance[from][spender]) {
```

Comment:
This bug in the contract was found by looking at the Anvil DEBUG console, and confirming that the code was indeed failing because of an overflow and not a custom error.

After changing this code, shows a custom error and its signature matches the custom error.

We can't detect custom errors _from another contract_ in tests now, so we have to do it manually for now.

Reply:
whoops